service/src/main/java/uk/gov/hmcts/reform/fpl/events/PartyAddedToCaseEvent.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class PartyAddedToCaseEvent extends CallbackEvent {
+    private final List<Element<Representative>> representativesToNotify;
```

Comment:
Thoughts on just having `List<Representative>`, the `unwrap(...)` function can convert a `List<Element<T>>` to `List<T>`

Reply:
done